pkg/util/query/query.go

```diff
@@ -78,3 +81,28 @@ func DescribeSchemaWithInternals(session gocqlx.Session) (DescribedSchema, error
 	}
 	return schema, nil
 }
+
+// RaftReadBarrier performs read barrier on a random node to which session is connected to.
+// When used with single host session, it can be used to ensure that given node
+// has already applied all raft commands present in the raft log.
+func RaftReadBarrier(session gocqlx.Session) error {
+	// As this functionality is not available from Scylla API,
+	// SM uses the workaround described in https://github.com/scylladb/scylladb/issues/19213.
+	// Read barrier is performed before any schema change, so it's enough to try to apply
+	// some dummy schema changing CQL statement.
+	// In order to avoid misleading errors, it's good to use the IF EXISTS clause.
+	// Still, it might happen that SM does not have the permissions to perform dummy schema change.
+	// As described in the issue, this type of error still results in performing read barrier,
+	// so SM should treat it as if everything went fine.
```

Comment:
Please extend the comment with information that Scylla is expected to extend the API in one of the upcoming releases, so that this workaround in SM is not forgotten and can be removed at some point in time.
`TODO: use API call instead of DROP TABLE call` is one of the options.